settings.py

```diff
@@ -48,3 +48,5 @@
 #  or flows have been deleted.
 TIME_RECENT_DELETED_FLOWS = 60
 TIME_RECENT_UPDATED = 60
+
+TABLE_GROUP_ALLOWED = {'base', 'EVPL', 'EPL'}
```

Comment:
In the blueprint and spec you specified [`evpl` and `epl` as lower case](https://github.com/kytos-ng/of_multi_table/blob/master/openapi.yml#L178-L179), enums are case sensitive so it would lead to issues, can you fix this?

Reply:
Changed

Reply:
Looks good. @Alopalao, can you open a task to also update [`of_multi_table` openapi.yml](https://github.com/kytos-ng/of_multi_table/blob/master/openapi.yml#L177) removing `base` from `mef_eline` there too? If one day we ever need `base` for `mef_eline` we can introduce back, I don't think we'll ever need though.

Reply:
Coming next commit.